web/static/js/components/surveys/SurveyWizardRetryAttempts.jsx

```diff
@@ -177,7 +177,7 @@ class SurveyWizardRetryAttempts extends Component {
                 />
               </InputWithLabel>
               <span className='small-text-bellow'>
-                {t('Enter a delay like 5m, 3h or 1d to express a time unit')}
+                {t('Enter delays like 10m 2h 1d to express time units (default 1h, use values greater than 10m)')}
```

Comment:
The fallback delay is one single delay. That's why it was phrased differently.

Reply:
True! Thanks!